### PR TITLE
Merge rotate functions for uint8 and int8 images

### PR DESCRIPTION
**Pull request by lucianomartin**
The new function has been tested manually using [test_rotate_90 ](https://github.com/xmos/lib_camera/tree/lib_camera_v2/tests/hw_tests/test_rotate_90).

Part of https://xmosjira.atlassian.net/browse/XV-418

### CODE REVIEW

File: lib_camera/api/camera_conv.h [hunk 1]
```diff
@@ -119,50 +119,30 @@ void camera_int8_to_uint8(
...
-
-/**
- * @brief Rotates an RGB image 90 degrees clockwise 
+ * @brief Rotates an RGB image 90 degrees clockwise
  * assumes both src and dst images are previously allocated
  * Input image dimensions are [height][width][channel]
  * and output image dimensions are [width][height][channel]
  * Input image pointer can't be reused for the output image
- * 
+ *
```
Comment (xalbertoisorna) [inline lib_camera/api/camera_conv.h:134]: could you add a comment avoid input type saying only int8 or uint8 are supported?

**Review by xalbertoisorna — APPROVED** [review]
looks good! only one minor coment about docstring, other than that is good 👍

File: lib_camera/api/camera_conv.h (hunk 1 above)
Comment (lucianomartin) [reply lib_camera/api/camera_conv.h:136]: I added some info in https://github.com/xmos/lib_camera/pull/91/commits/238f5c1788de81dd4580fc28babeabe64090bfab.
Is that ok?

Comment (xalbertoisorna) [reply lib_camera/api/camera_conv.h:136]: good!

**Pull request merged**